manifests/config.pp

```diff
@@ -11,6 +11,7 @@
         'metrics'       => $winlogbeat::metrics,
         'event_logs'    => $winlogbeat::event_logs_final,
       },
+      'cloud'             => $winlogbeat::cloud,
```

Comment:
I'm not seeing this section in the template
or do we have general treatment for that?

Reply:
either way, having a test that proves this code correct would be useful.

Reply:
I see what you mean. Adding the erb template changes to set a cloud.id and cloud.auth and will get it working with my elastic.co version today